bundles/com.espressif.idf.core/src/com/espressif/idf/core/IDFDynamicVariables.java

```diff
@@ -0,0 +1,24 @@
+package com.espressif.idf.core;
+
+/**
+ * Class to contain all the variables that can be resolved by the IDFDynamicVariableResolver.java
+ * @author Ali Azam Rana
+ *
+ */
+public interface IDFDynamicVariables
+{
+	/**
+	 * idf.py resolver variable
+	 */
+	String IDF_PY = "IDF_PY"; //$NON-NLS-1$
```

Comment:
should be final

Reply:
I will change it to enum but variables in interfaces are public static and final by default.